deploy/load-test/stack/chat/app.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.4'
+
+services:
+    chat-follow-event-store:
+        image: marein/php-gaming-website:php-fpm
+        command: bin/console chat:follow-event-store pointer --select-all-subscribers
+        env_file: ../app.env
+        volumes:
+            - proxysql.sock:/var/run/proxysql
+        deploy:
+            placement:
+                constraints:
+                    - "node.labels.long-running==1"
+    chat-command:
+        image: marein/php-gaming-website:php-fpm
+        command: bin/console gaming:consume-messages -c chat.command
```

Comment:
The way to call this command changed with #185. That needs to be adapted to all consumers through this pull request.